helix-core/src/main/java/org/apache/helix/model/InstanceConfig.java

```diff
@@ -329,6 +330,17 @@ public long getInstanceEnabledTime() {
     return _record.getLongField(InstanceConfigProperty.HELIX_ENABLED_TIMESTAMP.name(), -1);
   }
 
+  public void setInstanceOperation(InstanceConstants.InstanceOperation operation) {
+    // TODO: also setInstanceEnabled after sanity check.
+
+    _record.setSimpleField(InstanceConfigProperty.INSTANCE_OPERATION.toString(),
```

Comment:
use InstanceConfigProperty.INSTANCE_OPERATION.name(). 

Not use toString anymore. It is not consistent with line 341